.github/workflows/nuget_push.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+
+[ "${DEBUG:-false}" = "true" ] && set -x
+
+help_message="\
+Usage:
+
+Runs 'dotnet package' and 'dotnet nuget push'.
+
+VERSION_NUMBER: The version number for the nuget.
+  PROJECT_FILE: The project file for the nuget.
+     NUGET_KEY: The Key to the Nugets. It's in the name."
+
+if [[ -z "$VERSION_NUMBER" ]]; then
+    echo "Missing or empty VERSION_NUMBER environment variable." >&2
+    echo "$help_message"
+    exit 1
+fi
+
+if [[ -z "$PROJECT_FILE" ]]; then
+    echo "Missing or empty PROJECT_FILE environment variable." >&2
+    echo "$help_message"
+    exit 1
+fi
+if [[ -z "$NUGET_KEY" ]]; then
+    echo "Missing or empty NUGET_KEY environment variable." >&2
+    echo "$help_message"
+    exit 1
+fi
+
+sanitized_version_number=${VERSION_NUMBER//\+/.}
+echo $sanitized_version_number
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)